src/TabNine.ts

```diff
@@ -87,8 +91,12 @@ export class TabNine {
   }
 
   private static runTabNine(inheritStdio : boolean = false, additionalArgs: string[] = []): child_process.ChildProcess {
+    const ext = TabNineExtension.getInstance();
     const args = [
       "--client=vscode",
+      "--client-metadata",
+      "clientVersion=" + vscode.version,
+      "pluginVersion=" + ext.version,
```

Comment:
@dimacodota maybe we also one if this is an insiders build or not?

Reply:
we will get this as part of the `vscode.version`, if it is insiders the version will be `-insiders`